README.md

```diff
@@ -14,18 +14,18 @@ The EKS Anywhere package controller and the EKS Anywhere Curated Packages (refer
 
 ---
 
```

Comment:
I'm not a big fan of "The" here. It works for "The Batman", but not "The Terry Howe"

Reply:
I agree. This is the original approved verbiage though, so I don't want to mess up with it. I can update the rest of the README file to not use "the" convention. Does that work?

Reply:
Leave the disclaimer as it is, but we don't need the "the".

Reply:
Updated all the references except the disclaim itself.